kotlin/src/test/kotlin/org/axonframework/extensions/kotlin/CommandGatewayExtensionsTest.kt

```diff
@@ -41,7 +35,11 @@ internal class CommandGatewayExtensionsTest {
     fun `Send extension should invoke correct method on the gateway`() {
         every { subjectGateway.send(exampleCommand, any<CommandCallback<ExampleCommand, Any>>()) } just Runs
 
-        subjectGateway.send<ExampleCommand, Any>(command = exampleCommand, onError = { _, _, _ -> }, onSuccess = { _, _, _ -> })
```

Comment:
I'd consider keeping this call with explicit generics parameters and infered types in the callbacks, both as a check for type inference and a mini example of possible usage.

Otherwise, this test would be the same as the other one.

Reply:
Please try this after #37 is merged. I think it is not compiling anymore.